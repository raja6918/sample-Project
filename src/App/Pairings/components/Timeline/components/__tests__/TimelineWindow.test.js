import React from 'react';
import { mount } from 'enzyme';
import { TimelineWindow } from '../';

const minProps = {
  calendarData: [
    {
      days: [
        {
          dayIndex: 0,
          dayName: 'Wednesday',
          dayNumber: '25',
          dayShortName: 'We',
          isCarryIn: true,
          isCarryOut: false,
          isDayAfterPeriodEnd: false,
          isFirstDayOfMonth: false,
          isPeriodEnd: false,
          isPeriodStart: false,
          isWeekend: false,
          relativeDay: -7,
        },
        {
          dayIndex: 1,
          dayName: 'Thursday',
          dayNumber: '26',
          dayShortName: 'Th',
          isCarryIn: true,
          isCarryOut: false,
          isDayAfterPeriodEnd: false,
          isFirstDayOfMonth: false,
          isPeriodEnd: false,
          isPeriodStart: false,
          isWeekend: false,
          relativeDay: -6,
        },
      ],
      daysCount: 7,
      monthName: 'September',
      monthNumber: '8',
      year: '2018',
    },
  ],
  columnWidthInPx: 40,
  timelineVisibleDays: 5,
  onScrollX: jest.fn(),
  timelineWindowHeight: 100,
  id: 10,
  hourWidthInPx: 40,
  timelineWindowRef: () => {},
};

test.skip('TimelineWindow Component renders', () => {
  expect(mount(<TimelineWindow {...minProps} />)).toMatchSnapshot();
});
