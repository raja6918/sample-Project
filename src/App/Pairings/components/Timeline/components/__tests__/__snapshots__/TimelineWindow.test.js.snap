// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimelineWindow Component renders 1`] = `
<TimelineWindow
  calendarData={
    Array [
      Object {
        "days": Array [
          Object {
            "dayIndex": 0,
            "dayName": "Wednesday",
            "dayNumber": "25",
            "dayShortName": "We",
            "isCarryIn": true,
            "isCarryOut": false,
            "isDayAfterPeriodEnd": false,
            "isFirstDayOfMonth": false,
            "isPeriodEnd": false,
            "isPeriodStart": false,
            "isWeekend": false,
            "relativeDay": -7,
          },
          Object {
            "dayIndex": 1,
            "dayName": "Thursday",
            "dayNumber": "26",
            "dayShortName": "Th",
            "isCarryIn": true,
            "isCarryOut": false,
            "isDayAfterPeriodEnd": false,
            "isFirstDayOfMonth": false,
            "isPeriodEnd": false,
            "isPeriodStart": false,
            "isWeekend": false,
            "relativeDay": -6,
          },
        ],
        "daysCount": 7,
        "monthName": "September",
        "monthNumber": "8",
        "year": "2018",
      },
    ]
  }
  columnWidthInPx={40}
  hourWidthInPx={40}
  id={10}
  innerWidth={0}
  onScrollX={[MockFunction]}
  outerWidth={0}
  timelineVisibleDays={5}
  timelineWindowHeight={100}
  timelineWindowRef={[Function]}
>
  <ScrollBar
    className=""
    component="div"
    containerRef={[Function]}
    onScrollX={[MockFunction]}
  >
    <div
      className="scrollbar-container "
    >
      <div
        className="pt-window-container"
        id="pt-window-10"
        style={
          Object {
            "height": 100,
            "width": 0,
          }
        }
      >
        <div
          className="pt-window"
          style={
            Object {
              "width": 0,
            }
          }
        >
          <TimelineColumns
            columnWidthInPx={40}
            days={
              Array [
                Object {
                  "dayIndex": 0,
                  "dayName": "Wednesday",
                  "dayNumber": "25",
                  "dayShortName": "We",
                  "isCarryIn": true,
                  "isCarryOut": false,
                  "isDayAfterPeriodEnd": false,
                  "isFirstDayOfMonth": false,
                  "isPeriodEnd": false,
                  "isPeriodStart": false,
                  "isWeekend": false,
                  "relativeDay": -7,
                },
                Object {
                  "dayIndex": 1,
                  "dayName": "Thursday",
                  "dayNumber": "26",
                  "dayShortName": "Th",
                  "isCarryIn": true,
                  "isCarryOut": false,
                  "isDayAfterPeriodEnd": false,
                  "isFirstDayOfMonth": false,
                  "isPeriodEnd": false,
                  "isPeriodStart": false,
                  "isWeekend": false,
                  "relativeDay": -6,
                },
              ]
            }
            hourWidthInPx={40}
            rowsContainerRef={
              Object {
                "current": <div
                  class="pt-horizontal-lines"
                >
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                </div>,
              }
            }
            timelineVisibleDays={5}
          >
            <div
              className="pt-vertical-lines"
            >
              <TimelineDay
                className="pt-vl carry-day"
                hourWidthInPx={40}
                key="0"
                style={
                  Object {
                    "height": "100%",
                    "width": 40,
                  }
                }
                timelineVisibleDays={5}
              >
                <div
                  className="pt-vl carry-day "
                  style={
                    Object {
                      "height": "100%",
                      "width": 40,
                    }
                  }
                />
              </TimelineDay>
              <TimelineDay
                className="pt-vl carry-day"
                hourWidthInPx={40}
                key="1"
                style={
                  Object {
                    "height": "100%",
                    "width": 40,
                  }
                }
                timelineVisibleDays={5}
              >
                <div
                  className="pt-vl carry-day "
                  style={
                    Object {
                      "height": "100%",
                      "width": 40,
                    }
                  }
                />
              </TimelineDay>
            </div>
          </TimelineColumns>
          <TimelineRows
            rows={
              Array [
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
                Object {},
              ]
            }
            rowsContainerRef={
              Object {
                "current": <div
                  class="pt-horizontal-lines"
                >
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                  <div
                    class="pt-hl"
                    style="height: 40px;"
                  />
                </div>,
              }
            }
          >
            <div
              className="pt-horizontal-lines"
            >
              <div
                className="pt-hl"
                key="0"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="1"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="2"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="3"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="4"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="5"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="6"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="7"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="8"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="9"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="10"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="11"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="12"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="13"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="14"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="15"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="16"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="17"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="18"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="19"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="20"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="21"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="22"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="23"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="24"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="25"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="26"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="27"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="28"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="29"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="30"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="31"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="32"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="33"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
              <div
                className="pt-hl"
                key="34"
                style={
                  Object {
                    "height": 40,
                  }
                }
              />
            </div>
          </TimelineRows>
        </div>
      </div>
    </div>
  </ScrollBar>
</TimelineWindow>
`;
